Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/Dog_generated.cs

```diff
@@ -0,0 +1,288 @@
+﻿// <auto-generated />
+using SourceGeneratorPlayground;
+using SourceGeneratorPlayground.Generated;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+using System.Xml.Serialization;
+using System.Reflection;
+using System.ComponentModel;
+using Realms;
+using Realms.Weaving;
+using Realms.Schema;
+
+namespace SourceGeneratorPlayground
+{
+    [Generated]
+    [Woven(typeof(DogObjectHelper))]
+    public partial class Dog : IRealmObject, INotifyPropertyChanged, IReflectableType
+    {
+        public static ObjectSchema RealmSchema = new ObjectSchema.Builder("Dog", ObjectSchema.ObjectType.RealmObject)
+        {
+            Property.Primitive("Name", RealmValueType.String, isPrimaryKey: false, isIndexed: false, isNullable: true, managedName: "Name"),
+            Property.Object("Owner", "Person", managedName: "Owner"),
+        }.Build();
+
+        #region IRealmObject implementation
+
+        private IDogAccessor _accessor;
+
+        IRealmAccessor IRealmObjectBase.Accessor => Accessor;
+
+        internal IDogAccessor Accessor => _accessor = _accessor ?? new DogUnmanagedAccessor(typeof(Dog));
```

Comment:
Shouldn't this be:
```suggestion
        internal IDogAccessor Accessor => _accessor ?? (_accessor = new DogUnmanagedAccessor(typeof(Dog)));
```

It's almost the same, but I find it a little clearer.